ipaplatform/base/services.py

```diff
@@ -55,7 +55,8 @@
                      'rpcbind', 'kadmin', 'sshd', 'autofs', 'rpcgssd',
                      'rpcidmapd', 'pki_tomcatd', 'chronyd', 'domainname',
                      'named', 'ods_enforcerd', 'ods_signerd', 'gssproxy',
-                     'nfs-utils', 'sssd', 'NetworkManager']
+                     'nfs-utils', 'sssd', 'NetworkManager',
+                     'ipa_custodia', 'ipa_dnskeysyncd','ipa_otpd']
```

Comment:
DNSSEC also has ``ipa-ods-exporter.socket``.

Reply:
The whole service list needs a review but that's separate from the current PR.
I've opened https://pagure.io/freeipa/issue/7925 to that regard.